src/components/Header.test.js

```diff
@@ -0,0 +1,21 @@
+import Header from './Header'
+import { render } from '@testing-library/react'
+import user from '@testing-library/user-event'
+
+describe('Header', () => {
+  it('shows stop button if paused, show correct ', () => {
```

Comment:
```suggestion
  it('shows stop button if appState is paused', () => {
```
and doesn't it also render a 'stop' button?